modules/pre-commit.nix

```diff
@@ -381,26 +381,28 @@ in
           elif ! ${git}/bin/git rev-parse --git-dir &> /dev/null; then
             echo 1>&2 "WARNING: pre-commit-hooks.nix: .git not found; skipping installation."
           else
+            GIT_WC=`${git}/bin/git rev-parse --git-dir`/..
+
             # These update procedures compare before they write, to avoid
             # filesystem churn. This improves performance with watch tools like lorri
             # and prevents installation loops by via lorri.
 
-            if readlink .pre-commit-config.yaml >/dev/null \
-              && [[ $(readlink .pre-commit-config.yaml) == ${configFile} ]]; then
+            if readlink "''${GIT_WC}/.pre-commit-config.yaml" >/dev/null \
+              && [[ $(readlink "''${GIT_WC}/.pre-commit-config.yaml") == ${configFile} ]]; then
               echo 1>&2 "pre-commit-hooks.nix: hooks up to date"
             else
               echo 1>&2 "pre-commit-hooks.nix: updating $PWD repo"
 
-              [ -L .pre-commit-config.yaml ] && unlink .pre-commit-config.yaml
```

Comment:
This should probably do what it did before, to unlink the file for a migration path.